### ISSUE TITLE
James 1:21-22 missing a space

### ISSUE THREAD

**Issue by saibotsivad** [TehShrike/world-english-bible#1]
Looking at James 1:21-22 the output JSON doesn't have a space at the end of 21 like it should: https://github.com/TehShrike/world-english-bible/blob/master/json/james.json#L193

The raw HTML of that section looks like:

```html
which is able to save your souls.<a href="#FN2" class="notemark">†<span class="popup"> or, preserve your life.</span></a>  <span class="verse" id="V22">22&#160;</span>But be doers of the word
```

**Comment by TehShrike** [comment]
The space is missing in the intermediate JSON form, too, so the space must have been lost during the HTML parsing.

**Comment by TehShrike** [comment]
Okay, I finally think I got everything tracked down.  I pushed up a bunch of commits that fix some general whitespace issues.

Random whitespace elements are now processed, and inside of stanzas, there are now separate "line text" and "line break" objects, instead of just "line" objects.  The old data model was insufficient because there are plenty of places where verses in stanzas would span multiple lines, like in [Deborah's song](http://ebible.org/web/JDG05.htm).

The changes have been published as 1.0.0.
